lib/dynamic-bootstrap.js

```diff
@@ -28,7 +28,7 @@ function getEnv(opts) {
   return bootstrapEnv;
 }
 
-function buildCode(opts) {
+async function buildCode(opts) {
   if (opts.code) return opts.code;
 
   var env = getEnv(opts);
```

Comment:
it looks like you're still perhaps in the middle of converting this? sorry, i may be missing some context :-)

but let's change all the `var`s to `let`s